README.md

```diff
@@ -219,3 +219,18 @@ MaxText supports training and inference of various open models. Follow user guid
 
 * [Gemma](https://ai.google.dev/gemma): a family of open-weights Large Language Model (LLM) by [Google DeepMind](https://deepmind.google/), based on Gemini research and technology. \
 You can run decode and finetuning using instructions mentioned [here](https://github.com/google/maxtext/blob/main/getting_started/Run_Gemma.md).
+
+## Deterministic Data Input Pipeline - Grain
+
+MaxText users can optionally use [Grain](https://github.com/google/grain?tab=readme-ov-file), a deterministic data input pipeline. Grain makes the training process reproducible, disruption-proof, and debuggable. To use this pipeline:
```

Comment:
Can you say clearly the main advantage?  (That we checkpoint the state of the iterator!)

Reply:
Great point! How does it look now?